### ISSUE TITLE
Match between sources

### ISSUE THREAD

**Issue by franciscogouveia** [franciscogouveia/hapi-rbac#29]
Example:

``` js
{
    'credentials:documents': { field: 'params:document_id' }
}
```

See: https://github.com/franciscogouveia/rbac-core/issues/21

**Comment by juanjoLenero** [comment]
@franciscogouveia  Will this be implemented? Any pointers for someone wanting to implement it? I need this for a project so I might do it.

**Comment by franciscogouveia** [comment]
Available in version `2.3.0`
